contracts/RenPool.sol

```diff
@@ -178,8 +178,8 @@ contract RenPool {
     function fulfillWithdrawRequest(address _target) external {
         address sender = msg.sender;
         uint amount = withdrawRequests[_target];
-        // ^ This could not be defined plus make sure amount > 0
-        // TODO: make sure user cannot fullfil his own request
+        require(amount > 0, "Amount has to be positive");
+        require(sender != _target, "Sender cannot be yourself");
         // TODO: add test for when _target doesn't have an associated withdrawRequest
```

Comment:
I think the third todo is automatically implemented when we test if the amount is positive, what do you think?

Reply:
Not sure if I follow. What I was trying to say is: in case we have this situation `_target doesn't have an associated withdrawRequest` then the code should revert. I think your code implements this situation. 

Now, what I was referring was to add an actual test to the test suit to cover this case

Reply:
Oh right, I thought you meant checking it in the smart contract, my bad. 
Will definitely implement the test yes